velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -120,6 +193,15 @@ void WindowTestBase::testWindowFunction(
       assertQuery(queryInfo.planNode, queryInfo.querySql);
     }
   }
+
+  for (const auto& overClause : overClauses) {
```

Comment:
Adding this here would almost double test times and window tests are already very expensive. We should separate streaming for spark based testing I feel. Please add a separate method for streamingwindow based testing and add new tests in the Spark velox/functions/sparksql/window/tests folder for these.

Reply:
Add StreamingWindowTest in velox/functions/sparksql/window/tests.